configure

```diff
@@ -583,6 +618,12 @@ while [[ ! -z $1 ]] ; do
 #      HDF5LIB="-I$VALUE/include $VALUE/lib/libhdf5_hl.a $VALUE/lib/libhdf5.a -lm"
 #      echo "Using HDF5LIB: $HDF5LIB"
 #      ;;
+    "--with-readline")
```

Comment:
Fair enough about the headers. I think cpptraj only uses 2 or 3 functions from readline, so you could just extern declare their prototypes even without having the headers. This is common for BLAS.